ansible/main.yml

```diff
@@ -48,6 +48,20 @@
   roles:
     - { role: okfn-org-manager, tags: ['heroku-backup'] }
 
+- hosts: s138.okserver.org
+  sudo: true 
```

Comment:
That would not work for my file structure.  I don't mind specifying it when I run a playbook.  It's not much of a hardship.